org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle

```diff
@@ -15,6 +15,9 @@ jar {
 	from('model') {
 		into('model')
 	}
+	from(sourceSets.main.allSource) {
+		include '**/*.xtext'
+	}
```

Comment:
Then it would be more correct to move the xtext file to src/main/resources. But ok, let's keep your solution so we don't change the current behavior too much.